rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationBasedOnParentClassMethodRector/Fixture/extended_class_narrowed.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationBasedOnParentClassMethodRector\Source\SomeClassWithReturnMixed;
+
+class MyClass extends SomeClassWithReturnMixed
+{
+    public function run(): string
+    {
+    }
+}
+
+?>
+-----
```

Comment:
after part is not needed if both equal, also please add `skip_` prefix to fixture name, ref for example

https://github.com/rectorphp/rector-src/blob/db7012edc116426e441645abdc1cffcb8c47ee85/rules-tests/Php80/Rector/ClassMethod/AddParamBasedOnParentClassMethodRector/Fixture/skip_no_parent_method.php.inc#L5

Reply:
Well this test not when there is no parent method but when existing return type is narrowed parnt return type. So i removed unnecessary after part and renamed it to skip_narrowed